src/python/grpcio_tests/tests/interop/methods.py

```diff
@@ -44,25 +44,48 @@
 from src.proto.grpc.testing import messages_pb2
 from src.proto.grpc.testing import test_pb2
 
+_INITIAL_METADATA_KEY = "x-grpc-test-echo-initial"
+_TRAILING_METADATA_KEY = "x-grpc-test-echo-trailing-bin"
+
+# If the request came with metadata in the context, this
+# helper function will copy it to the returned metadata.
+def MaybeEchoMetaData(context):
```

Comment:
Prefer PEP-8 names; should this be `_maybe_echo_metadata`? (And do the same for every behavior name introduced in this change that is free to be changed; some are not because they must match what's found in the `.proto` file.)

Also in gRPC vocabulary "metadata" is one word, so no need for capital letter or underscore to separate "meta" from "data".